website/_sass/custom/custom.scss

```diff
@@ -0,0 +1,24 @@
+// We should override $nav-width(-md), however this breaks code highlighting and other styles.
+// This seems to be an issue for following the recommended approach.
+// See https://github.com/just-the-docs/just-the-docs/issues/982.
```

Comment:
No, this will continue to work. When that bug gets fixed, we can adhere to the official/recommended approach of creating a custom scheme and overriding `$nav-width` s.t. that param gets used throughout the rest of the styling.

See https://just-the-docs.github.io/just-the-docs/docs/customization/#custom-schemes

Reply:
Check. Didn't fully grok that page yet, but pushed a commit with a proposal :)